test/functional/legacy/025_jump_tag_hidden_spec.lua

```diff
@@ -25,12 +23,17 @@ describe('jump to a tag with hidden set', function()
     feed_command('set hidden')
 
     -- Create a link from test25.dir to the current directory.
-    feed_command('!rm -f test25.dir')
-    feed_command('!ln -s . test25.dir')
+    if helpers.iswin() then
+      feed_command('!rd /q/s test25.dir')
```

Comment:
`helpers.rmdir()` instead

Reply:
It can't handle directory junctions in Windows and luafilesystem does not support Windows symlinks yet. Best to call `delete()` through nvim or os.execute() and raise an exception if it fails.

Reply:
In https://github.com/neovim/neovim/commit/6cbf290d56d618601635d301969c697cfcc2b653#diff-33008990d305461ece636fb5f89a6b7bL442, why is there no check for link mode?